src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.service;
+
+import java.time.LocalDate;
+import java.util.List;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionService {
+
+    MovieSession add(MovieSession movieSession);
+
+    MovieSession get(Long id);
```

Comment:
The `get()` method should return an `Optional<MovieSession>` instead of `MovieSession`. This is important for handling potential null values safely, as specified in the task requirements.